### PR TITLE
=doc Add dependency for Akka Typed #23222

### PR DESCRIPTION
**Pull request by jlprat**
Issue: #23222
- Add dependency for Akka Typed in docs
- Make page dual with Scala and Java
- Minor styling fixes

### CODE REVIEW

File: akka-docs/src/main/paradox/java/typed.md [hunk 1]
```diff
@@ -9,16 +9,49 @@ This module is currently marked as @ref:[may change](common/may-change.md) in th
...
+Make sure that you have the following dependency in your project:
+
+Scala
+:   @@@vars
+    ```
+    "com.typesafe.akka" %% "akka-typed" % "$akka.version$"
+    ```
+    @@@
+
+Java
```
Comment (ktoso) [inline akka-docs/src/main/paradox/java/typed.md]: Scala (sbt) / Java (maven) - note that we say sbt/maven/gradle on http://akka.io/docs/ so we should start converging towards that :)

Should we also add a Gradle tab?

**Review by ktoso — COMMENTED** [review]
LGTM but let's add Gradle :)

**Comment by ktoso** [comment]
Thanks a ton :)

File: akka-docs/src/main/paradox/java/typed.md (hunk 1 above)
Comment (jlprat) [inline akka-docs/src/main/paradox/java/typed.md]: Done :)

**Review by ktoso — APPROVED** [review]
Great, thanks :)

File: akka-docs/src/main/paradox/java/typed.md [hunk 2]
```diff
@@ -9,16 +9,58 @@ This module is currently marked as @ref:[may change](common/may-change.md) in th
...
+    @@@
+
+## Introduction
+
+As discussed in @ref:[Actor Systems](general/actor-systems.md) (and following chapters) Actors are about
 sending messages between independent units of computation, but how does that
 look like? In all of the following these imports are assumed:
 
-@@snip [IntroSpec.scala]($akka$/akka-typed-tests/src/test/java/jdocs/akka/typed/IntroTest.java) { #imports }
+Scala
```
Comment (raboof) [inline akka-docs/src/main/paradox/java/typed.md]: Thanks for splitting this out already - this'll make replacing the Java version with a symlink much easier later

File: akka-docs/src/main/paradox/scala/typed.md [hunk 3]
```diff
@@ -243,7 +305,7 @@ This particular `main` Actor is created using `Actor.deferred`, which is like a
 Creation of the behavior instance is deferred until the actor is started, as opposed to `Actor.immutable`
 that creates the behavior instance immediately before the actor is running. The factory function in 
 `deferred` pass the `ActorContext` as parameter and that can for example be used for spawning child actors.
-This ``main`` Actor creates the chat room and the gabbler and the session between them is initiated, and when the
+This `main` Actor creates the chat room and the gabbler and the session between them is initiated, and when the
```
Comment (raboof) [inline akka-docs/src/main/paradox/scala/typed.md]: :+1:

**Review by raboof — APPROVED** [review]

**Pull request merged**